scripts/copyPackageContent.ps1

```diff
@@ -0,0 +1,27 @@
+# Checks that expected files are present & have contents after the publish process to the local cache
+param(
+  [Parameter(Mandatory=$true)][string] $ArtifactId,
+  [Parameter(Mandatory=$true)][string] $PackageSegment,
+  [Parameter(Mandatory=$true)][string] $Version,
+  [Parameter(Mandatory=$true)][string] $OutputDirectory,
+  [Parameter()][string] $GroupId = "com.microsoft.kiota",
+  [Parameter()][string] $ComponentsSegment = "..\components",
+  [Parameter()][string] $buildOutDir = "build\publishing-repository"
+)
+
+#components\bundle\build\publishing-repository\com\microsoft\kiota\microsoft-kiota-bundle\1.8.5\
+
+$groupIdPath = $GroupId -replace "\.", [System.IO.Path]::DirectorySeparatorChar
+$packageFullPath = Join-Path -Path $PSScriptRoot -ChildPath $ComponentsSegment -AdditionalChildPath $PackageSegment, $buildOutDir, $groupIdPath, $ArtifactId, $Version
+
+Write-Output "---------------------------------------------------"
+Write-Output "Copying package contents at $packageFullPath"
+
+if(-not (Test-Path -Path $packageFullPath)) {
+  Write-Output "Package not found in local cache."
+  exit 1
+}
+
+Copy-Item -Path "$packageFullPath\*" -Destination $OutputDirectory
```

Comment:
@baywet, are we expecting only zip files directly within `$packageFullPath\*`, with no subdirectories present? If there are subdirectories in this path, we should include the `-Recurse` parameter with `Copy-Item` to ensure all contents, including those in subfolders, are copied.

Reply:
it's a flat structure, this is due to how the java module packaging works, and I don't believe it'll change anytime soon